test/utils/prepare.ts

```diff
@@ -90,6 +91,18 @@ export default function prepare (t: Test, pkg?: Object) {
         throw err
       }
     },
+    loadModules: async () => {
+      try {
+        return await loadYamlFile<any>('node_modules/.modules.yaml') // tslint:disable-line
+      } catch (err) {
+        if (err.code === 'ENOENT') return null
+        throw err
+      }
+    },
+    rewriteDependencies: async (deps) => {
+      pkgJson = Object.assign(pkgJson, { dependencies: deps })
```

Comment:
you can use the spread operator in typescript. `{...pkgJson, dependencies}`

Reply:
The compiler complaints about something not being assignable if I use the spread operator :/